### PR TITLE
email: add domain and url to email notification

### LINKED ISSUE #33: mail notifications do not include the proper url to the map
No clickable url in the text, no map name in the text or information has how to regenerate it.
```email
The XCSoar Map Generator has finished your map.
It can be downloaded at /download?uuid=c17dae5e-474c-11ec-8dec-525400c7f7bf
This link is valid for 7 days.
```

### PR DESCRIPTION
**Pull request by lordfolken**
closes #33

### CODE REVIEW

File: lib/xcsoar/mapgen/server/worker.py [hunk 1]
```diff
@@ -24,10 +25,10 @@ def __send_download_mail(self, job):
 To: {to}
 Subject: XCSoar Map Generator - Download ready ({name}.xcm)
 
-The XCSoar Map Generator has finished your map.
-It can be downloaded at {url}
+The XCSoar Map Generator has finished your map: {name}
+It can be downloaded at {protocol}://{domain}{url}
 This link is valid for 7 days.
-'''.format(to=job.description.mail, name=job.description.name, url=job.description.download_url)
+'''.format(to=job.description.mail, name=job.description.name, protocol=mapgen['protocol'],domain=mapgen['domain'],url=job.description.download_url)
```
Comment (rawtaz) [inline lib/xcsoar/mapgen/server/worker.py:31]: I would put spaces between the arguments here.

**Review by rawtaz — COMMENTED** [review]
Besides the cosmetics I mentioned I think it looks good :)

File: lib/xcsoar/mapgen/server/worker.py (hunk 1 above)
Comment (rawtaz) [reply lib/xcsoar/mapgen/server/worker.py:31]: But I guess I see your point in not doing so, keeping those related ones together. Not convinced it improves readability though :)

Comment (lordfolken) [reply lib/xcsoar/mapgen/server/worker.py]: Uniformity makes sense. Thx.

**Pull request merged**